_data/projects/EigengrausGenerator.yml

```
@@ -12,7 +12,7 @@ tags:
 - generation
 - tabletop
 upforgrabs:
-  name: good first issue
+  name: help wanted
   link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/good%20first%20issue
```

To keep this in sync with the label change:

```suggestion
   link: https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/labels/help%20wanted
```

---

Ah thanks, missed that